lib/queuePopulator/LogReader.js

```diff
@@ -355,7 +355,22 @@ class LogReader {
         });
     }
 
+    _isSupportedOperation(record) {
```

Comment:
NIT: I don't remember the standard. Since this method will likely be overwritten by the Zenko/Mongo log reader class, should we remove the private prefix: `_` ?